gravitee-am-service/src/main/java/io/gravitee/am/service/http/WebClientBuilder.java

```diff
@@ -125,6 +144,85 @@ private void setProxySettings(WebClientOptions options, String url) {
         }
     }
 
+    private void setSSLSettings(WebClientOptions options) {
+        if (isSSLEnabled) {
+            options.setTrustAll(isSSLTrustAllEnabled);
```

Comment:
Do we have to define : 
* the list of accepted cipher suite
* the list of TLS version 
:thinking:  ?